repository-data/site/src/main/resources/hcm-config/hst/configurations/intranet/sitemap.yaml

```diff
@@ -2,6 +2,46 @@
 definitions:
   config:
     /hst:hst/hst:configurations/intranet/hst:sitemap:
+      /_any_:
```

Comment:
What's the difference between `/_any_/_index_` ? Wouldn't just `/_index_` have sufficed? I thought `_index_` matches any number of nodes

Reply:
In this case, index will match folders and redirect them to a document at 'folder-path'/content. I can remove the index node if its decided we dont want that behaviour

Reply:
no, I think it makes sense.